packages/docs/timetable-planner/README.md

```diff
@@ -48,169 +51,109 @@ When preferences are made, those times will not be changed.
-                "FRIDAY":[32400, 36000],
-                "size":160,
-                "enrolment":0
-
-            }],
-            "L0107":[{
-                "WEDNESDAY":[64800, 75600],
-                "size":160,
-                "enrolment":0
-                }]
-        },
-        "CSC318H5F2019":{
-            "L0101":[{
-                "TUESDAY":[68400, 75600],
-                "WEDNESDAY":[68400, 75600],
-                "size": 60,
-                "enrolment": 0
-            }],
-            "T0107":[{
-                "THURSDAY":[68400, 75600],
-                "size": 60,
-                "enrolment": 0
-                }]
-        },
-        
-}
+The algorithm starts out basic, it checks if the sections given have conflict to each other. If there is no conflict, the algorithm will return "valid timetable", or if there is a conflict between any section, it will return "invalid timetable".
 
-```
+Based on the validity checking, the algorithm starts to be able to add constraints to the timetable.
 
-#### Output:
-
-```json
-
-{
-    "MONDAY":[
-        {
-            "CSC108H5F2019L0101":[32400, 36000]
-        }
-        ],
-    "TUESDAY":[
-        {
-            "CSC318H5F2019L0101":[68400, 75600]
-        }
-        ],
-    "WEDNESDAY":[
-        {
-            "CSC108H5F2019L0101":[32400, 36000]
-        },
-        {
-            "CSC318H5F2019L0101":[68400, 75600]
-        }
-        ],
-    "THURSDAY":[
-        {
-            "CSC318H5F2019T0101":[68400, 75600]
-        }
-        ],
-    "FRIDAY":[
-        {
-            "CSC108H5F2019L0101":[32400, 36000]
-        }
-        ],
-}
+The algorithm then takes in an invalid time argument that inputs from the user's preference time off. This argument is being treated like a course which will be parsed in to the timetable and check for conflict as well.
 
-```
+The algorithm then implements an idle time function which takes in a set of valid timetable and returns the max and the min idle time depend on the user's preference.
 
-## Current Implementateion
-### (WIP)
-
-Given a set of times, check if there is a valid timetable avaliable.
-
-Outputs "Valid" if such timetable exist, or else "inValid".
-
-#### Input:
-
-```json
-#provided one set of time to check if a valid timetable exist
-{
-    "CourseCode": {
-        "CSC108H5F2019":{
-            "L0101":[{
-                "MONDAY":[32400, 36000],
-                "WEDNESDAY":[32400, 36000],
-                "FRIDAY":[32400, 36000],
-                "size":160,
-                "enrolment":0
-
-            }],
-        },
-        "CSC318H5F2019":{
-            "L0101":[{
-                "TUESDAY":[68400, 75600],
-                "WEDNESDAY":[68400, 75600],
-                "size": 60,
-                "enrolment": 0
-            }],
-        },
-        
+The algorithm then stores the user's locked courses in a list, with future optimization, the locked courses in the list will stays the same section.
```

Comment:
The algorithm then stores the user's locked courses in a list _which will stay in position despite future optimizations._